src/styles/_catalog.scss

```
@@ -14,9 +14,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform 300ms;
+    transition-duration: 300ms;
```

```suggestion
    transition-duration: 300ms;
    transition-property: transform, color;
```
Потрібно вказувати, до якої властивості буде застосовуватись transition